tiledb/sm/query/result_tile.cc

```diff
@@ -484,6 +466,87 @@ void ResultTile::compute_results_dense(
   }
 }
 
+template <>
+void ResultTile::compute_results_sparse<char>(
+    const ResultTile* result_tile,
+    unsigned dim_idx,
+    const Range& range,
+    std::vector<uint8_t>* result_bitmap) {
+  auto coords_num = result_tile->cell_num();
+  auto range_start = range.start_str();
+  auto range_start_size = (uint64_t)range_start.size();
+  auto range_end = range.end_str();
+  auto range_end_size = (uint64_t)range_end.size();
+  auto& r_bitmap = (*result_bitmap);
+
+  // Get coordinate tile
+  const auto& coord_tile = result_tile->coord_tile(dim_idx);
+
+  // Get offset buffer
+  const auto& coord_tile_off = std::get<0>(coord_tile);
+  void* buff_off_v = nullptr;
+  assert(coord_tile_off.chunked_buffer()->nchunks() == 1);
+  coord_tile_off.chunked_buffer()->internal_buffer(0, &buff_off_v);
+  auto buff_off = (const uint64_t*)buff_off_v;
```

Comment:
The chunked buffers for coordinate tiles are always contiguous, but they may have more than one logical chunk. Let's make these lines:
```
assert(
  coord_tile_off.chunked_buffer()->buffer_addressing() ==
    ChunkedBuffer::BufferAddressing::CONTIGUOUS);
const uin64_t* buff_off =
  static_cast<uint64_t*>(coord_tile_off.chunked_buffer()->get_contiguous_unsafe());
```

Similarly below. Note that `get_contiguous_unsafe` just bypasses the check that we have in the `assert`, so there is no reason to use the safe variant `get_contiguous`.